app/components/sidebar.tsx

```diff
@@ -167,10 +163,11 @@ function useGlobalShortcut() {
   };
 
   useEffect(() => {
-    handleMasks();
-    handleSettings();
-    handleNewChat();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
+    if (window.__TAURI__) {
```

Comment:
you still need to put `// eslint-disable-next-line react-hooks/exhaustive-deps` for fix/ignore stupid react/nextjs warning

Reply:
Thanks for your patience! All changes are cleaned and submitted.